Deliverable2/Program.cs

```diff
@@ -4,79 +4,76 @@
-        AddPadding(Padding);
+        AddPadding(padding);
 
         //Warn Guests About Group Size Limit//
         Console.ForegroundColor = ConsoleColor.Red;
         Console.WriteLine("                     ****Please note**** \n" +
             "Due to current staffing shortages we must limit party size to 6 guests or less");
-        AddPadding(Padding);
+        AddPadding(padding);
         Console.ForegroundColor = ConsoleColor.White;
 
         //Get Number Of Guests In The Group//
 
         Console.WriteLine("How many guests are in your party?");
-        GuestCount = int.Parse(Console.ReadLine());
+        var guestCount = int.Parse(Console.ReadLine());
 
         //Do Not Continue Until Valid Group Size Is Entered//
-        while (!ReEnterGroupSize)
+        while (!reEnterGroupSize)
         {
-            AddPadding(Padding);
-            if (GuestCount > 6)
+            AddPadding(padding);
+            if (guestCount > 6)
             {
-                Console.WriteLine("I'm sorry, your party of " + GuestCount + " exceeds our current limit of 6 guests per party");
-                AddPadding(Padding);
+                Console.WriteLine("I'm sorry, your party of " + guestCount + " exceeds our current limit of 6 guests per party");
+                AddPadding(padding);
 
                 //Give User Chance To Adjust Group Size || Leave//
-                Console.WriteLine("Would you like to adjust your party size or find a different resturant? \n" + "Please enter Yes or y to adjust your party size or press any other key to leave");
-                AddPadding(Padding);
+                Console.WriteLine("Would you like to adjust your party size or find a different restaurant? \n" + "Please enter Yes or y to adjust your party size or press any other key to leave");
+                AddPadding(padding);
 
-                string GroupResponse = Console.ReadLine();
+                var groupResponse = Console.ReadLine();
 
                 //Handle User Capitalization Variance//
-                GroupResponse = GroupResponse.ToLower();
+                groupResponse = groupResponse.ToLower();
 
-                if (GroupResponse == "yes" || GroupResponse == "y")
+                if (groupResponse == "yes" || groupResponse == "y")
                 {
-                    ReEnterGroupSize = false;
+                    reEnterGroupSize = false;
                     Console.Clear();
                     Console.WriteLine("Please enter a new party size that is 6 or less guests");
-                    GuestCount = int.Parse(Console.ReadLine());
+                    guestCount = int.Parse(Console.ReadLine());
                 }
                 else
                 {
                     Console.WriteLine("Sorry You Are Leaving, Please Stop By Again. \n" + "Have a nice day!");
-                    ReEnterGroupSize = true;
-                    DoNotRunReceipt = true;
+                    return; // completes right here.
```

Comment:
Rather than setting a variable to exit, this function, we can just `return` here.